testoption.go

```diff
@@ -9,11 +9,11 @@ import (
 // TestOption applies optional settings to a test.
 type TestOption func(*testOptions)
 
-// WithStartTime returns a test option that sets the time of the test runner's
-// clock at the start of the test.
+// StartVirtualClockAt returns a test option that sets initial the time of the
```

Comment:
```suggestion
// StartVirtualClockAt returns a test option that sets the initial time of the
```

Reply:
I've since changed this again on `main` - I changed this function name again to `StartTimeAt()` to match the language with `AdvanceTime()`.